### PR TITLE
fix(image): use seeded picsum

### LINKED ISSUE #1964: some urlPicsumPhotos are broken
example https://picsum.photos/id/725/640/480

Unfortunately picsum.photos appears to have some gaps in the 1...1000 range

See https://github.com/DMarby/picsum-photos/issues/83

We could blacklist the missing images?

### PR DESCRIPTION
**Pull request by Shinigami92**
closes #1964

### CODE REVIEW

**Review by ST-DDT — DISMISSED** [review]

**Review by xDivisionByZerox — DISMISSED** [review]

File: src/modules/image/index.ts [hunk 1]
```diff
@@ -228,9 +228,7 @@ export class ImageModule {
   ): string {
     const { width = 640, height = 480, grayscale = false, blur } = options;
 
-    let url = `https://picsum.photos/id/${this.faker.number.int(
-      1000
-    )}/${width}/${height}`;
+    let url = `https://picsum.photos/seed/${this.faker.number.int()}/${width}/${height}`;
```
Comment (matthewmayer) [inline src/modules/image/index.ts]: The seed values seem a bit long maybe could do like faker.string.alphanumeric with a fixed length of 4 or 5?

**Review by matthewmayer — CHANGES_REQUESTED** [review]

File: src/modules/image/index.ts (hunk 1 above)
Comment (xDivisionByZerox) [reply src/modules/image/index.ts]: Whats the problem with "long" values? 
We are talking about a difference of 15/16 or 4/5 characters...

Comment (matthewmayer) [reply src/modules/image/index.ts]: they're just a bit ugly. no big deal :D

Comment (ST-DDT) [reply src/modules/image/index.ts]: **Team Decision**

- The length of the link should be variable
- We will use alphaNumeric with mixed case

**Review by ST-DDT — DISMISSED** [review]

File: src/modules/image/index.ts [hunk 2]
```diff
@@ -189,12 +189,12 @@ export class ImageModule {
    * @param options.blur Whether the image should be blurred. Defaults to `false`.
    *
    * @example
-   * faker.image.urlPicsumPhotos() // 'https://picsum.photos/id/241/640/480'
-   * faker.image.urlPicsumPhotos({ width: 128 }) // 'https://picsum.photos/id/241/128/480'
-   * faker.image.urlPicsumPhotos({ height: 128 }) // 'https://picsum.photos/id/241/640/128'
-   * faker.image.urlPicsumPhotos({ grayscale: true }) // 'https://picsum.photos/id/241/640/480?grayscale'
-   * faker.image.urlPicsumPhotos({ blur: 4 }) // 'https://picsum.photos/id/241/640/480?blur=4'
-   * faker.image.urlPicsumPhotos({ blur: 4, grayscale: true }) // 'https://picsum.photos/id/241/640/480?grayscale&blur=4'
+   * faker.image.urlPicsumPhotos() // 'https://picsum.photos/seed/241/640/480'
```
Comment (matthewmayer) [inline src/modules/image/index.ts]: Should also have alphanumeric seeds in the examples

**Review by matthewmayer — CHANGES_REQUESTED** [review]

File: src/modules/image/index.ts [hunk 3]
```diff
@@ -189,12 +189,12 @@ export class ImageModule {
...
+   * faker.image.urlPicsumPhotos({ width: 128 }) // 'https://picsum.photos/seed/241/128/480'
+   * faker.image.urlPicsumPhotos({ height: 128 }) // 'https://picsum.photos/seed/241/640/128'
+   * faker.image.urlPicsumPhotos({ grayscale: true }) // 'https://picsum.photos/seed/241/640/480?grayscale'
+   * faker.image.urlPicsumPhotos({ blur: 4 }) // 'https://picsum.photos/seed/241/640/480?blur=4'
+   * faker.image.urlPicsumPhotos({ blur: 4, grayscale: true }) // 'https://picsum.photos/seed/241/640/480?grayscale&blur=4'
```
Comment (Shinigami92) [inline src/modules/image/index.ts]:
```suggestion
   * faker.image.urlPicsumPhotos() // 'https://picsum.photos/seed/NWbJM2B/640/480'
   * faker.image.urlPicsumPhotos({ width: 128 }) // 'https://picsum.photos/seed/NWbJM2B/128/480'
   * faker.image.urlPicsumPhotos({ height: 128 }) // 'https://picsum.photos/seed/NWbJM2B/640/128'
   * faker.image.urlPicsumPhotos({ grayscale: true }) // 'https://picsum.photos/seed/NWbJM2B/640/480?grayscale'
   * faker.image.urlPicsumPhotos({ blur: 4 }) // 'https://picsum.photos/seed/NWbJM2B/640/480?blur=4'
   * faker.image.urlPicsumPhotos({ blur: 4, grayscale: true }) // 'https://picsum.photos/seed/NWbJM2B/640/480?grayscale&blur=4'
```

Comment (matthewmayer) [reply src/modules/image/index.ts]: They shouldn't be the same seed for all?

**Review by ST-DDT — APPROVED** [review]

File: src/modules/image/index.ts (hunk 3 above)
Comment (ST-DDT) [reply src/modules/image/index.ts]: Having the same seed here probably makes it easier to see the differences between each invocation.

**Review by matthewmayer — APPROVED** [review]

**Pull request merged**